bash.html.markdown

```diff
@@ -25,7 +25,21 @@ Nearly all examples below can be a part of a shell script or executed directly i
 # As you already figured, comments start with #. Shebang is also a comment.
 
 # Simple hello world example:
-echo Hello world!
+echo "Hello world!"
```

Comment:
Careful here. The `!"` at the end will be interpreted as history substitution. Use single quotes here, or I would just leave the line as is, since it works as intended.